ui/tests/acceptance/access/methods-test.js

```diff
@@ -3,74 +3,68 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import { currentRouteName, click } from '@ember/test-helpers';
+import { currentRouteName, click, find, findAll, visit } from '@ember/test-helpers';
 import { clickTrigger } from 'ember-power-select/test-support/helpers';
 import { module, test } from 'qunit';
 import { setupApplicationTest } from 'ember-qunit';
 import { setupMirage } from 'ember-cli-mirage/test-support';
-import { create } from 'ember-cli-page-object';
-import page from 'vault/tests/pages/access/methods';
-import authEnable from 'vault/tests/pages/settings/auth/enable';
-import authPage from 'vault/tests/pages/auth';
-import ss from 'vault/tests/pages/components/search-select';
-import consoleClass from 'vault/tests/pages/components/console/ui-panel';
-
 import { v4 as uuidv4 } from 'uuid';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { mountAuthCmd, runCmd } from 'vault/tests/helpers/commands';
+import { login } from 'vault/tests/helpers/auth/auth-helpers';
 
-const consoleComponent = create(consoleClass);
-const searchSelect = create(ss);
+const { searchSelect } = GENERAL;
 
 module('Acceptance | auth-methods list view', function (hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
 
   hooks.beforeEach(function () {
     this.uid = uuidv4();
-    return authPage.login();
+    return login();
   });
 
   test('it navigates to auth method', async function (assert) {
-    await page.visit();
+    await visit('/vault/access/');
     assert.strictEqual(currentRouteName(), 'vault.cluster.access.methods', 'navigates to the correct route');
-    assert.ok(page.methodsLink.isActive, 'the first link is active');
-    assert.strictEqual(page.methodsLink.text, 'Authentication Methods');
+    assert.dom('[data-test-sidebar-nav-link="Authentication Methods"]').hasClass('active');
   });
 
   test('it filters by name and auth type', async function (assert) {
     assert.expect(4);
     const authPath1 = `userpass-1-${this.uid}`;
     const authPath2 = `userpass-2-${this.uid}`;
     const type = 'userpass';
-    await authEnable.visit();
-    await authEnable.enable(type, authPath1);
-    await authEnable.visit();
-    await authEnable.enable(type, authPath2);
-    await page.visit();
-    // filter by auth type
+    await visit('/vault/settings/auth/enable');
+    await runCmd(mountAuthCmd(type, authPath1));
+    await visit('/vault/settings/auth/enable');
+    await runCmd(mountAuthCmd(type, authPath2));
+    await visit('/vault/access/');
 
+    // filter by auth type
     await clickTrigger('#filter-by-auth-type');
-    await searchSelect.options.objectAt(0).click();
```

Comment:
this test was originally failing because we were arbitrarily selecting the first item in the auth type dropdown. in this taste I basically just updated it to specifically select `userpass` - while here decided to remove `ember-cli-page-object` methods 🧹 
<img width="873" alt="Screenshot 2024-10-16 at 11 32 49 AM" src="https://github.com/user-attachments/assets/bb372d5b-add6-4fe8-a037-7d6a2a8e9aa7">